### ISSUE TITLE
`./vland --self-test` fails to run

### ISSUE THREAD

**Issue by ArdvarkDev** [parke/vland#3]
the self-test prints the following out
```
$ vland --self-test

vland  self_test  ...

self_test_clean
```

no idea what might be causing this

**Comment by parke** [comment]
Thank you for the report.  It is a bug in line 820.

https://github.com/parke/vland/blob/main/vland#L820

The current line is:  
` [ -d "$vland_dir" ]  ||  return`

I believe the fix will be:  
`[ -d "$vland_dir" ]  ||  return  0`

When the directory at path `$vland_dir` does not exist, the above line currently returns 1.  
Then, because I have `set -o errexit`, the script silently exits.  
This is one of the risks of using `||` instead of an `if` clause when `errexit` is set.

Fyi, I have made many improvements to `vland` since I the version I published on `20211229`.  Among these improvements is/will be a completely rebuilt self-test.  I expect to publish the next version of `vland` within the next week, and I expect this bug to be fixed in that version.  Indeed, I even expect the improved self-test to expose and fail on this bug, even on my development system.

The reason the current self-test failed to expose this bug to me is that `$vland_dir` almost always exists on my system.
